deploy/gk-deploy

```diff
@@ -561,9 +578,10 @@ if [[ ${GLUSTER} -eq 1 ]] && [[ ${LOAD} -eq 0 ]]; then
     exit 1
   fi
   output "OK"
+  EXISTS_GLUSTERFS=1
 fi
 
-if [[ ${LOAD} -eq 0 ]]; then
+if [[ ${EXISTS_DEPLOY_HEKETI} -eq 0 ]] && [[ ${EXISTS_HEKETI} -eq 0 ]]; then
   sed -e "s/\${HEKETI_EXECUTOR}/${EXECUTOR}/" -e "s/\${SSH_PORT}/${SSH_PORT}/" -e "s/\${SSH_USER}/${SSH_USER}/" -e "s/\${SSH_SUDO}/${SSH_SUDO}/" heketi.json.template > heketi.json
```

Comment:
That just creates the template, you later have to "process" the template to generate the final yaml that defines the actual resources.